app/controllers/problems_searcher.rb

```diff
@@ -25,6 +25,10 @@ module ProblemsSearcher
       Array(Problem.find(err_ids))
     end
 
+    expose(:selected_problems_ids) do
+      selected_problems.map(&:id).map(&:to_s)
```

Comment:
This is probably fine, but I'm curious about why we need to_s here. Do you know why?